tests/schemas/api_v3/transaction.schema.js

```diff
@@ -30,6 +30,17 @@ const block = {
 	height: Joi.number().integer().min(1).required(),
 	timestamp: Joi.number().integer().positive().max(getCurrentTime())
 		.required(),
+	isFinal: Joi.boolean().required(),
+};
+
+const transactionMetaReceipientSchema = {
+	address: Joi.string().pattern(regex.ADDRESS_BASE32).required(),
```

Comment:
Can we please rename it to `ADDRESS_LISK32` everywhere instead `ADDRESS_BASE32`
```suggestion
	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
```

Reply:
Applied.